stripe/_api_requestor.py

```diff
@@ -434,6 +507,52 @@ def request_raw(
 
         return rcontent, rcode, rheaders, my_api_key
 
+    async def request_raw_async(
+        self,
+        method: str,
+        url: str,
+        params: Optional[Mapping[str, Any]] = None,
+        supplied_headers: Optional[Mapping[str, str]] = None,
+        is_streaming: bool = False,
+        api_mode: Optional[Literal["preview", "standard"]] = None,
+        *,
+        _usage: Optional[List[str]] = None,
+    ) -> Tuple[object, int, Mapping[str, str], str]:
+        abs_url, headers, post_data, my_api_key = self._get_request_raw_args(
+            method, url, params, supplied_headers, is_streaming, api_mode
+        )
+
+        if is_streaming:
+            (
+                rcontent,
+                rcode,
+                rheaders,
+            ) = await self._client_async.request_stream_with_retries(
+                method, abs_url, headers, post_data, _usage=_usage
+            )
+        else:
+            (
+                rcontent,
+                rcode,
+                rheaders,
+            ) = await self._client_async.request_with_retries(
+                method, abs_url, headers, post_data, _usage=_usage
+            )
+
+        _util.log_info(
```

Comment:
can this me shared?

Reply:
[share logging](https://github.com/stripe/stripe-python/pull/1165/commits/ced75f85c5e0b54449eea8cf74958b52be1e82df)